solarnet/playbooks/domains.yaml

```diff
@@ -0,0 +1,108 @@
+---
+- hosts: jenkins
+  vars:
+      do:
+          DO_API_KEY: "{{ lookup('env','DIGITAL_OCEAN_API_KEY') }}"
+          DO_CLIENT_ID: "{{  lookup('env','DIGITAL_OCEAN_CLIENT_ID') }}"
+  tasks:
+    - name: "install dopy"
+      pip: name=dopy
+    - environment: do
+      digital_ocean_domain:
+        state: present
+        name: uranus.nyc.srv.protocol-dev.com
```

Comment:
we probably want to move off of `protocol-dev.com` for this and to `i.ipfs.io` -- do we need the full `"<host>.<city>.srv."` prefix? (not sure where it came from originally).